lib/prop/limiter.rb

```diff
@@ -134,22 +136,28 @@ def handles
 
       private
 
-      def at_threshold?(mark, threshold)
-        mark >= threshold
-      end
-
-      def disabled?
-        !!@disabled
-      end
-
       def prepare(handle, key, params)
```

Comment:
prepare is called on each call ... but the callsite should not know about the strategy ... that should be in the setup ?